.github/dependabot-copy-my-content.yml

```diff
@@ -0,0 +1,22 @@
+version: 2
+updates:
+- package-ecosystem: github-actions
+  directory: "/"
+  schedule:
+    interval: daily
+    time: '04:00'
+  open-pull-requests-limit: 10
+- package-ecosystem: gradle
+  directory: "/"
+  schedule:
+    interval: daily
+    time: '04:00'
+  ignore:
+    - dependency-name: "com.squareup.okhttp3:okhttp"
```

Comment:
Is it connected to gradle?

Reply:
Yes, it least it can interpret t more or less. In principle, it should understand gradle ranges as well, but I haven't dared trying them. Let's try that once it's working.

Reply:
Yes, at least dependabot can partially interpret gradle. It 's WIP, I think.